inputs/mvs_config.json

```diff
@@ -474,54 +434,50 @@
                 },
                 "unit": "kW"
             },
-            "output power": {
+            "storage capacity": {
                 "age_installed": {
                     "unit": "year",
                     "value": 0
                 },
                 "capex_fix": {
                     "unit": "currency",
-                    "value": 0.0
-                },
-                "capex_var": {
-                    "unit": "currency/kWh",
                     "value": 0
                 },
-                "c_rate": {
-                    "unit": "factor of total capacity (kWh)",
-                    "value": 1
+                "capex_var": {
+                    "unit": "currency/unit",
+                    "value": 4000
                 },
                 "efficiency": {
                     "unit": "factor",
-                    "value": 0.9500000000000001
+                    "value": 0
                 },
                 "installedCap": {
-                    "unit": "kWh",
+                    "unit": "unit",
                     "value": 0
                 },
-                "label": "discharging_power",
+                "label": "storage capacity",
```

Comment:
Of the labels we should choose storage capacity - I've seen that this was changed in dev.

Reply:
@Piranias introduced this in #259. Right now, spaces are used.

Reply:
Also, unneeded parameters like "crate" for storage capacity are not parsed anymore (#259)

Reply:
> I wonder what is the convention here "storage_capacity" "storageCapacity" "storage capacity"...

For labels I think it is nicer to have spaces, for variable names/field names we should normally only use snake case (`_`) as the language is python and not javascript (which adopts lower camel case). I think we can make the change of the variable/fields name prior to a release on master to not create too many errors :)

Reply:
I thought we could do two releases for that - one keeping above names, another also changing "capex_var" into something like "c_specific"/"specific_investment_costs" and so on... (comp. #40)